RecoEgamma/EgammaTools/interface/tables.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RecoEgamma_EgammaTools_tables_h
+#define RecoEgamma_EgammaTools_tables_h
+
+#include "FWCore/SOA/interface/Column.h"
+#include "FWCore/SOA/interface/Table.h"
+#include "RecoEgamma/EgammaTools/interface/LazyResult.h"
+
+#include <vector>
+
+namespace egamma {
+
+  namespace soa {
+
+    namespace col {
+
+      SOA_DECLARE_COLUMN(Pt, float, "pt");
+      SOA_DECLARE_COLUMN(Eta, float, "eta");
+      SOA_DECLARE_COLUMN(Phi, float, "phi");
+      SOA_DECLARE_COLUMN(Vz, float, "vz");
+
+    }  // namespace col
```

Comment:
Couldn't address all comments in one go, I suddenly had connection problems.

Yeah I was also thinking about this. I think you are right about splitting the columns from the generic tables, after all we already have `EleTkIsolFromCands` which is using the columns but not the table defined here.

However, I think anticipating different column categories is a bit premature at this point, and even then I'm probably against splitting it up, mainly to avoid collisions in `egm::soa::col` (current name). Similar for the tables: we might have tables building on each other with for example the `AddColumns_t` mechanism [1]. Having different tables in different files that are connected by added and subtracted columns would be also complicated to oversee.

Would you  be fine with one file for the columns, and one for the tables in `CommonTools/Utils`, while keeping more generic names? I understand and would share the feeling that simply `tables.h` and `columns.h` sounds too generic, would you prefer `SoATables.h` and `SoAColumns.h`?

[1] https://github.com/cms-sw/cmssw/blob/master/FWCore/SOA/test/table_t.cppunit.cpp#L69

Reply:
I'd like to avoid a setup where a single header is expected to describe everything for a concept of SOA.
So, some categorization from the start (like ```Kinematic*```) would make more sense to me.

Reply:
Okay!